docs/content/operations/including-extensions.md

```diff
@@ -45,17 +61,22 @@ extensions/
 │   ├── slf4j-log4j12-1.6.1.jar
 │   ├── snappy-java-1.1.1.6.jar
 │   ├── zkclient-0.3.jar
-│   └── zookeeper-3.4.7.jar
 └── mysql-metadata-storage
-    ├── jdbi-2.32.jar
     ├── mysql-connector-java-5.1.34.jar
-    └── mysql-metadata-storage-0.8.0-rc1.jar
+    └── mysql-metadata-storage-0.9.0.jar
 ```
 
-As you can see, under ```extensions``` there are two sub-directories ```druid-kafka-eight``` and ```mysql-metadata-storage```, each sub-directory denotes an extension that Druid might load.
+As you can see, underneath ```extensions``` there are two sub-directories ```druid-kafka-eight``` and ```mysql-metadata-storage```, 
+each sub-directory denotes an extension that Druid could load.
+
+**Tell Druid what extensions to load**
+
+Use `druid.extensions.loadList`(See [Configuration](../configuration/index.html) ) to specify a
+list of names of extensions that should be loaded by Druid.
 
-3) To have Druid load a specific list of extensions present under the root extension directory, set `druid.extensions.loadList` to the list of extensions to load. Using the example above, if you want Druid to load ```druid-kafka-eight``` and ```mysql-metadata-storage```, you can specify `druid.extensions.loadList=["druid-kafka-eight", "mysql-metadata-storage"]`.
+For example, `druid.extensions.loadList=["druid-kafka-eight", "mysql-metadata-storage"]` instructs Druid to load `druid-kafka-eight` 
+and `mysql-metdata-storage` extensions. That is, the name you specified in the list should be the same as its extension folder's name.
 
 If you specify `druid.extensions.loadList=[]`, Druid won't load any extensions from the file system.
 
-If you don't specify `druid.extensions.loadList`, Druid will load all the extensions under the root extension directory.
+If you don't specify `druid.extensions.loadList`, Druid will load all the extensions under the directory specified by `druid.extensions.directory`.
```

Comment:
jars on the classpath will always be loaded by Druid.
If druid.extensions.directory is not specified, it will be set to its default value, which is ```extensions``` underneath Druid's working directory. See [here](https://github.com/druid-io/druid/blob/master/docs/content/configuration/index.md)